content/network/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Network of Research Software Quality
+description: Network Information
+identifier: Network
+weight: 10
+showDate: false
+groupByYear: false
+---
+
+## Network of Research Software Quality ##
+
+The EVERSE project is committed to improving the quality of software in European research in the international context. EVERSE will launch a Network of Research Software Quality to allow us to work with individuals and partner organisations to achieve this common goal.
```

Comment:
"will launch" -> "is launching" to give more of a sense that this is happening now